src/main/java/frc/trigon/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,45 @@
+package frc.trigon.robot.subsystems.gripper;
+
+import com.ctre.phoenix6.controls.VoltageOut;
+import frc.trigon.robot.subsystems.MotorSubsystem;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+
+public class Gripper extends MotorSubsystem {
+    private final TalonFXMotor motor = GripperConstants.MOTOR;
+    private final VoltageOut voltageRequest = new VoltageOut(0).withEnableFOC(GripperConstants.FOC_ENABLED);
+
+    @Override
+    public void setBrake(boolean brake) {
+        motor.setBrake(brake);
+    }
+
+    @Override
+    public void drive(double targetDrivePower) {
+        motor.setControl(voltageRequest);
+    }
```

Comment:
1. wrong implementation, you're not using targetDrivePower
2. this subsystem doesn't have SysID since it doesn't have any gains, no need for this function